### PR TITLE
docs: add badges and Discord to README

### LINKED ISSUE #51: How do you want to engage with Puck?
The last few days have been a blast - we've gone from 8 stars on Tuesday morning to 2.1k and counting on Thursday.

We've been thinking a lot about the best way to engage with you all.

Currently we're considering:

1. GitHub only (issues, discussions and releases)
2. X/Twitter (via the [@hellomeasuredco account](https://twitter.com/hellomeasuredco), or a new dedicated account)
3. Discord
4. Mailing list

Are there any things we're missing, or anything you'd like us to consider? Free-form responses welcome.

### PR DESCRIPTION
**Pull request by chrisvxd**
Closes #51

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+    <img src="https://img.shields.io/badge/MADE%20BY%20Measured-000000.svg?style=for-the-badge&labelColor=000">
+  </a>
+  <a aria-label="NPM version" href="https://www.npmjs.com/package/@measured/puck">
+    <img alt="" src="https://img.shields.io/npm/v/@measured/puck.svg?style=for-the-badge&labelColor=000000">
+  </a>
+  <a aria-label="License" href="https://github.com/measuredco/blob/main/LICENSE">
+    <img alt="" src="https://img.shields.io/npm/l/@measured/puck.svg?style=for-the-badge&labelColor=000000">
+  </a>
+  <a aria-label="Join the community on Discord" href="https://discord.gg/D9e4E3MQVZ">
+    <img alt="" src="https://img.shields.io/badge/Join%20the%20Discord-blueviolet.svg?style=for-the-badge&logo=Discord&labelColor=000000&logoWidth=20">
```
Comment (chrisvxd) [inline README.md:14]: Alternatively we can use the Discord chat widget

image

File: README.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
 # puck
 
+<p align="left">
+  <a aria-label="Measured logo" href="https://measured.co">
+    <img src="https://img.shields.io/badge/MADE%20BY%20Measured-000000.svg?style=for-the-badge&labelColor=000">
+  </a>
+  <a aria-label="NPM version" href="https://www.npmjs.com/package/@measured/puck">
+    <img alt="" src="https://img.shields.io/npm/v/@measured/puck.svg?style=for-the-badge&labelColor=000000">
+  </a>
+  <a aria-label="License" href="https://github.com/measuredco/blob/main/LICENSE">
```
Comment (anglepoised) [inline README.md:10]:
```suggestion
  <a aria-label="License" href="https://github.com/measuredco/puck/blob/main/LICENSE">
```

File: README.md (hunk 1 above)
Comment (anglepoised) [reply README.md:14]: Think it makes sense to stick with what's there and maybe change to the live widget down the line, if take-up of Discord is good.

**Review by anglepoised — CHANGES_REQUESTED** [review]
A comment and a suggested fix for broken link.

File: README.md (hunk 2 above)
Comment (chrisvxd) [reply README.md]: Fixed!

**Review by anglepoised — APPROVED** [review]

**Pull request merged**